homeassistant/components/zwave/__init__.py

```diff
@@ -193,19 +193,19 @@ def _node_object_id(node):
     return node_object_id
 
 
-def _object_id(value):
+def object_id(value):
     """Return the object_id of the device value.
 
     The object_id contains node_id and value instance id
     to not collide with other entity_ids.
     """
-    object_id = "{}_{}_{}".format(slugify(_value_name(value)),
-                                  value.node.node_id, value.index)
+    _object_id = "{}_{}_{}".format(slugify(_value_name(value)),
+                                   value.node.node_id, value.index)
 
     # Add the instance id if there is more than one instance for the value
     if value.instance > 1:
         return '{}_{}'.format(object_id, value.instance)
```

Comment:
Thanks. I'm new to HA so will have to spend a while getting to grips with the codebase! Thanks for your co-operation. Should I open a new issue?